tests/container-e2e-tests-wsl2.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
```

Comment:
You had updated '#!/bin/bash' to '#!/usr/bin/env bash' elsewhere. Does this need to change too?

Reply:
The other file was an unintentional change. I will leave this one as-is and revert the other.